### PR TITLE
Add a S3 mock to be able to run S3 tests even on the CI

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,7 +27,13 @@ latest:
 
 .PHONY:
 test-inside: build
-	docker-compose run --rm -e TRAVIS wsgi scripts/launch_inside_tests.sh
+	docker run --rm -v $$PWD/data:/data c2corg/v6_images:latest rm -rf /data/incoming/* /data/active/* /data/.minio.sys/format.json /data/.minio.sys/multipart  /data/.minio.sys/tmp
+	docker-compose stop || true
+	docker-compose rm -f || true
```
Comment (gberaudo) [inline Makefile]: Why do you call `docker-compose rm`? Isn't it supposed to be handled above by the `--rm` parameter?

File: Makefile [hunk 2]
```diff
@@ -27,7 +27,13 @@ latest:
 
 .PHONY:
 test-inside: build
-	docker-compose run --rm -e TRAVIS wsgi scripts/launch_inside_tests.sh
+	docker run --rm -v $$PWD/data:/data c2corg/v6_images:latest rm -rf /data/incoming/* /data/active/* /data/.minio.sys/format.json /data/.minio.sys/multipart  /data/.minio.sys/tmp
```
Comment (gberaudo) [inline Makefile]: What is this line for?

Comment (pvalsecc) [reply Makefile]: If the previous run failed, this same line wasn't executed after the test and the bucket wasn't cleared.

File: Makefile (hunk 1 above)
Comment (pvalsecc) [reply Makefile]: the --rm deleted the wsgi container, but not the linked minio container.

**Comment by pvalsecc** [comment]
> Can the cleanups be done in the tests?

The tests are run inside a container and I've been highly criticized in the past for giving access to docker inside the test containers.

**Pull request merged**